app/src/main/java/com/example/pantrypal/activities/Search.java

```diff
@@ -3,11 +3,13 @@
 import static android.widget.Toast.makeText;
 
 import android.content.Intent;
+import android.net.Uri;
```

Comment:
There are some unused import here. Could you do Ctrl + Alt + O to clean them up?